docs/yaml-config.rst

```diff
@@ -135,7 +135,7 @@ used for building documentation.
 python.version
 ``````````````
 
-* Default: ``2.7``
+* Default: ``3.5``
```

Comment:
This will be difficult to express... The actual default value here is the one from the admin panel

Reply:
3.5 has some issues (#5051) we may want to default to 3.6 or even 3.7.x because

>  Python 3.6.8 is planned to be the last bugfix release for 3.6.x. Following the release of 3.6.8, we plan to provide security fixes for Python 3.6 as needed through 2021, five years following its initial release.

from https://www.python.org/downloads/release/python-368/

This is also a good excuse to upgrade our Docker images.

Reply:
With our current release of docker images:

* `3.0` will default to `py3.5`
* `4.0` to `py3.5`
* `5.0rc1` to `py3.6`

So, I think we are fine here.